webknossos/webknossos/client/_resumable/chunk.py

```diff
@@ -36,6 +39,7 @@ def resolve_chunk(
     if not exists_on_server:
         tries = 0
         while not _send_chunk(client, config, file, chunk):
+            time.sleep(RETRY_DELAY)
```

Comment:
I assume this is run on a separate thread, so the blocking doesn't prevent simultaneous requests?

Reply:
Yes :+1: